### DISCUSSION TITLE
Runtime Loading works in the Editor but not in the WebGL Build

### DISCUSSION THREAD

**Discussion by Djokix98** [atteneder/glTFast#539] [Q&A]
Hello!
I have some issues when trying to load the glb file at runtime. 
This is the code that im using, and the model is loaded correctly in playmode inside the Editor.
![Screenshot 2022-12-13 at 15 43 14](https://user-images.githubusercontent.com/90897165/207365153-18024fb0-fb14-42a1-9013-28eb6fd91675.png)

However, when i build the project as WebGL, loading gets stuck and i always get the null reference exception, no matter which file im trying to load.
<img width="388" alt="Screenshot 2022-12-13 at 15 41 43" src="https://user-images.githubusercontent.com/90897165/207365598-d4a97ba2-1f2c-4291-82a1-6d596aee2d86.png">

Thanks in advance!

**Comment by atteneder** [comment]
hi @Djokix98 ,

It's hard to guess without more information. Try to make a development build to get a detailed stack trace (see [webgl debugging documentation](https://docs.unity3d.com/Manual/webgl-debugging.html)).

Unfortunately you cannot debug in WebGL, but you can place `Debug.Log` at strategic places in your/my code to figure where exactly it fails.

hth
